getAppStrings method returns a map instead of an string

rgonalo opened the pull request:
With this change getAppStrings returns a Map<String, String>, making it easier to be used:
```
Map<String, String> strings = driver.getAppStrings();
String string_value = strings.get('string_id');
```

TikhomirovSergey commented:
:+1: 
I like it!
@Jonahss What's your opinion?

rompic commented:
+1

Jonahss commented:
this is a breaking change, so will require a major version release. I think it's good though.
Can you add a node to the changelog?

rgonalo commented:
I have added the note to the changelog in version 4.0.0 ¿it is ok the version number?

I have also added to the PR an additional commit with a new getAppStrings() method to search app strings in the specified file, that is already allowed by appium server: https://github.com/appium/appium/blob/cdadd16d216abbe9838ed48a37af7d0aaed67150/lib/server/controller.js#L1120

Jonahss commented:
Cool! Thanks.
